include/libspirv/libspirv.h

```diff
@@ -260,6 +260,7 @@ typedef enum spv_validate_options_t {
   SPV_VALIDATE_ID_BIT = SPV_BIT(2),
   SPV_VALIDATE_RULES_BIT = SPV_BIT(3),
   SPV_VALIDATE_SSA_BIT = SPV_BIT(4),
+  SPV_VALIDATE_INSTRUCTION_BIT = SPV_BIT(5) | SPV_VALIDATE_LAYOUT_BIT,
```

Comment:
@dneto0 is correct, but I think it would be a good idea to isolate the instruction checks from the layout checks for unit testing purposes.